src/components/expanded-state/AvailableNetworksv2.tsx

```diff
@@ -116,13 +116,7 @@ const AvailableNetworksv2 = ({
     [asset, goBack, isReadOnlyWallet, navigate, networks]
   );
 
-  const handlePressContextMenu = useCallback(
-    // @ts-expect-error ContextMenu is an untyped JS component and can't type its onPress handler properly
-    ({ nativeEvent: { actionKey: chainId } }) => {
-      convertAssetAndNavigate(chainId);
-    },
-    [convertAssetAndNavigate]
-  );
+  const handlePressContextMenu = useCallback((chainId: string) => convertAssetAndNavigate(+chainId), [convertAssetAndNavigate]);
```

Comment:
maybe (not sure if good idea) we could make `convertAssetAndNavigate` take string and convert inside, to not need the `useCallback`

Reply:
ah actually just notice `DropdownMenu` is not `memo`ed so the fn being stable or not does not make a difference, the useCallback is just overhead rn, don't think we can easily memo it since it takes `children`